arc/data/ear-claims-key-attestation-ok.json

```
@@ -0,0 +1,25 @@
+{
+  "eat_profile": "tag:github.com,2023:veraison/ear",
+  "iat": 1666529184,
+  "ear.verifier-id": {
+    "developer": "https://veraison-project.org",
+    "build": "vts 0.0.1"
+  },
+  "ear.raw-evidence": "NzQ3MjY5NzM2NTYzNzQK",
+  "submods": {
+    "PARSEC_TPM": {
+      "ear.status": "affirming",
+      "ear.trustworthiness-vector": {
+        "instance-identity": 2,
+        "executables": 2,
+        "hardware": 2
+      },
+      "ear.appraisal-policy-id":
+        "https://veraison.example/policy/1/60a0068d",
```

```suggestion
        "policy://parsec/60a0068d",
```
To be in line with: https://github.com/veraison/services/issues/101